spec/javascripts/components/story/note/note_spec.js

```diff
@@ -10,22 +10,46 @@ describe('<Note/>', () => {
     createdAt: '27/08/2018'
   };
 
-  describe('when user deletes a note', () => {
-    it('triggers onDelete callback', () => {
-      const onDeleteSpy = sinon.spy();
+  const setup = propOverrides => {
+    const onDeleteSpy = sinon.spy();
+  
+    const wrapper = shallow(
+      <Note
+        note={note}
+        onDelete={onDeleteSpy}
+        disabled={false}
+        {...propOverrides}
+      />
+    );
 
-      const wrapper = shallow(
-        <Note
-          note={note}
-          onDelete={onDeleteSpy}
-        />
-      );
+    const button = wrapper.find('.delete-note-button');
 
-      const button = wrapper.find('.delete-note-button');
+    return { wrapper, button, onDeleteSpy };
```

Comment:
can we rename `button` to `deleteButton` to make the tests that use it easier to understand? I got a little confused when I saw this test https://github.com/Codeminer42/cm42-central/pull/540/files#diff-5b2bbd29d2e9844431270fb14ac57c98R31

Reply:
Yeah that's nice